cmd/vclusterctl/cmd/create.go

```diff
@@ -80,7 +82,18 @@ vcluster create test --namespace test
 	cobraCmd.Flags().StringVar(&cmd.TemplateVersion, "template-version", "", "[PLATFORM] The vCluster platform template version to use")
 	cobraCmd.Flags().StringArrayVar(&cmd.Links, "link", []string{}, "[PLATFORM] A link to add to the vCluster. E.g. --link 'prod=http://exampleprod.com'")
 	cobraCmd.Flags().StringVar(&cmd.Params, "params", "", "[PLATFORM] If a template is used, this can be used to use a file for the parameters. E.g. --params path/to/my/file.yaml")
+	cobraCmd.Flags().StringVar(&cmd.Params, "parameters", "", "[PLATFORM] If a template is used, this can be used to use a file for the parameters. E.g. --parameters path/to/my/file.yaml")
 	cobraCmd.Flags().StringArrayVar(&cmd.SetParams, "set-param", []string{}, "[PLATFORM] If a template is used, this can be used to set a specific parameter. E.g. --set-param 'my-param=my-value'")
+	cobraCmd.Flags().StringArrayVar(&cmd.SetParams, "set-parameter", []string{}, "[PLATFORM] If a template is used, this can be used to set a specific parameter. E.g. --set-parameter 'my-param=my-value'")
+	cobraCmd.Flags().StringVar(&cmd.Description, "description", "", "[PLATFORM] The description to show in the platform UI for this virtual cluster")
+	cobraCmd.Flags().StringVar(&cmd.DisplayName, "display-name", "", "[PLATFORM] The display name to show in the platform UI for this virtual cluster")
+	cobraCmd.Flags().StringVar(&cmd.Team, "team", "", "[PLATFORM] The team to create the space for")
+	cobraCmd.Flags().StringVar(&cmd.User, "user", "", "[PLATFORM] The user to create the space for")
+	cobraCmd.Flags().BoolVar(&cmd.UseExisting, "use", false, product.Replace("[PLATFORM] If the platform should use the virtual cluster if its already there"))
+	cobraCmd.Flags().BoolVar(&cmd.Recreate, "recreate", false, "[PLATFORM] If enabled and there already exists a virtual cluster with this name, it will be deleted first")
+	cobraCmd.Flags().BoolVar(&cmd.CreateSpaceContext, "create-space-context", true, product.Replace("[PLATFORM] If the CLI should create a kube context for the space"))
+	cobraCmd.Flags().BoolVar(&cmd.SwitchSpaceContext, "switch-space-context", true, product.Replace("[PLATFORM] If the CLI should switch the current context to the new context"))
```

Comment:
Those product Replace calls are unnecessary.

Can we also drop the "space" mention here?